tests/sar_pipeline/test_full_aws_docker_run.py

```diff
@@ -1,5 +1,17 @@
-# test a full build and run
-# NOTE - requires sufficient CPU, RAM and Disk Memory
+"""
+Overview:
+This is test for a complete build and run of the project docker image
+It should be completed prior to every PR and release. A local run is generally
+required, given the need for credentials, sufficient CPU, RAM and Disk Memory.
```

Comment:
Are you able to provide a rough estimate of CPU, RAM and disk memory required for running a scene?